README.md

```diff
@@ -46,6 +46,10 @@ vagrant up --provision
 # OR
 vagrant provision
 ```
+**KNOWN ISSUES**
+If the command line gets stuck in  "ssh auth method: private key" or get  "time out while waiting for the machine to boot",
```

Comment:
Each issue must have a sub-heading and description.


Refer to markdown syntax: https://guides.github.com/features/mastering-markdown/#syntax

Reply:
ok i will refer to that.